packages/client/hmi-client/src/components/drilldown/tera-drilldown-section.vue

```diff
@@ -31,6 +36,15 @@ footer {
 	gap: 0.5rem;
 }
 
+header {
+	margin-top: 1em;
+	gap: 0.75rem;
+}
```

Comment:
If you are creating a class for this, please remove the tailwind classname.
Having two places to edit styling is difficult.